src/test/java/services/CsvWriterImplTest.java

```diff
@@ -0,0 +1,39 @@
+package services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.impl.CsvWriterImpl;
+import core.basesyntax.service.CsvFileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvWriterImplTest {
+    public static final String VALID_FILE_NAME = "src/test/java/resources/WriteFileTest.csv";
+    public static final String INVALID_FILE_NAME = "";
+    public static final String REPORT = "type,fruit,quantity\n"
+            + "b,banana,20";
```

Comment:
```suggestion
    public static final String REPORT = "type,fruit,quantity" + System.lineSeparator()
            + "b,banana,20";
```
Or you can even use a text block